Login object is not global, and came_from_checkout is not available a…

mikrdgz opened the pull request:
…s a property at all, so made changes to docs that relfect this and moved login.form.error to Other Objects

# [DEVDOCS-959](https://jira.bigcommerce.com/browse/DEVDOCS-959)

## What changed?
In audit of login, found that form.login is not global, and even when adding form to other template with front matter, this property cannot be pulled. Edited GLobal Objects to reflect this finding. The property came_from_checkout does not appear to be an available property at all in Stencil based on what I saw in the code. Moved form.login.error to Other Objects section

tatiana-perry commented on markdown/stencil-docs/reference-docs/other-objects-and-properties-overview.md:
```diff
@@ -2819,6 +2819,10 @@ https://github.com/bigcommerce/cornerstone/tree/master/templates/components/comm
   </tr>
+  <tr>
+  <td>error</td>
+  <td>BC-defined message to display when customer’s login fails</td>
```
We never use BC publicly. Always BigCommerce. If you do see it in use, fix it.

tatiana-perry replied:
I also noticed that the form object returns a more information. Please edit the object to include it or create a spin off ticket. 

```
  "forms": {
    "login": {
      "error": "",
      "recaptcha": {
        "enabled": 1,
        "markup": "<div class=\"g-recaptcha\" data-sitekey=\"6LccmasUAAAAAIRhScC9asOrH_rQblw06weNOzDI\"></div><br/>",
        "public_key": "6LdWfFgSABBAAAI83aRectJhbwidegZKk8PzWBltH"
      },
      "success": ""
    }
  },
```

tatiana-perry replied:
`http://localhost:3000/login.php?debug=bar`

mikrdgz commented:
Fixed BC and changed to BigCommerce, also created spinoff ticket https://jira.bigcommerce.com/browse/DEVDOCS-1137 so we can include new properties for captcha
